extensions/nodes/node_status.go

```diff
@@ -141,39 +143,31 @@ func AllNodeDeleted(client *rancher.Client, ClusterID string) error {
 	return err
 }
 
-// IsNodeReplaced is a helper method that will loop and check if the node matching its type is replaced in a cluster.
-// It will return an error if the node is not replaced after set amount of time.
-func IsNodeReplaced(client *rancher.Client, oldMachineID string, clusterID string, numOfNodesBeforeDeletion int) (bool, error) {
-	numOfNodesAfterDeletion := 0
-	isOldMachineDeleted := true
-
+// IsNodeDeleted is a helper method that will loop and check if the node is deleted in the cluster.
+func IsNodeDeleted(client *rancher.Client, nodeName, ClusterID string) error {
 	ctx := context.Background()
 	err := wait.PollUntilContextTimeout(
-		ctx, oneSecondInterval, PollTimeout, true, func(ctx context.Context) (bool, error) {
```

Comment:
is PollTimeout still used anywhere? If not they should be removed

Reply:
That is used in function `Isv1NodeConditionMet` that isn't being touched in this PR at all.